### PR TITLE
Output Teacher Overview + Appearance

### PR DESCRIPTION
**Pull request by mestrona3**
Alyssa created the output teacher overview and I made some changes to it so that it is readable.

Some minor fixes were also needed to fix errors (i.e. just use cohortNames.size())

To test:
- run model
- open flamborough sheet, navigate to last output (just created)
- check to make sure text is wrapped, top row with day names is bolded and all model results expected (i.e. cohort name and subject appear in matrix), teacher names as column
- make sure that 'prep' and 'away' are listed in each applicable cell, without any slashes or other information

### CODE REVIEW

**Comment by mccurdy42** [comment]
I have run it 3x and a solution is found on eclipse, but the output is not creating a new sheet in excel and is not printing to the console.

**Review by ajacobs12 — DISMISSED** [review]
I was able to get this running on my local computer. That said, I think there are some issues with what is being printed. Classes 8b and 8c are actual cohorts but I think are being looked at as the dummy cohorts for prep and away. In the schedule, they are not showing a subject as assigned to them. Only the cohort name. See picture below: 
test

The dummy cohorts needed to be added to the cohort names array list in order for the actual subject to be printed. It also looks like the size of n3 was changed to just be cohortnames.size but it needs to have the extra 2 added to account for the additional dummy cohorts. Pic below:
image

While these changes are being committed, can we also fix the merge conflicts? That way once this is resolved, the PR is able to be merged right away!

File: FYDP_ModV1/src/version1.java [hunk 1]
```diff
@@ -268,6 +276,7 @@ public static void modelConfig() throws IOException {
         	
         	if (inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartCol+k).getCellType() == CellType.STRING) {
         		gradeNames.add(inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartCol+k).getStringCellValue());
+        		cohortNames.add(inputSheet2.getRow(cohortNameStartRow).getCell(gradeNameStartCol+k).getStringCellValue());
```
Comment (ahopkins19) [inline FYDP_ModV1/src/version1.java]: instead of gradeNameStartCol should be cohortNameStartCol

File: FYDP_ModV1/src/version1.java [hunk 2]
```diff
@@ -267,7 +275,8 @@ public static void modelConfig() throws IOException {
         	}
         	
         	if (inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartCol+k).getCellType() == CellType.STRING) {
-        		gradeNames.add(inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartCol+k).getStringCellValue());
+        		gradeNames.add(inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartRow+k).getStringCellValue());
```
Comment (ahopkins19) [inline FYDP_ModV1/src/version1.java]: gradeNames.add(inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStart**Row**+k).getStringCellValue());
this should be gradeNameStartCol in the .getCell() part

**Review by mccurdy42 — APPROVED** [review]
I have run this branch and it runs in an acceptable range and with the desired output

**Pull request closed without merging**